src/main/java/com/google/sps/data/GivrUser.java

```diff
@@ -104,22 +104,38 @@ public static void updateUserInDatastore(String identifyingProperty, String iden
     datastore.put(entity);
   }
 
+  public static GivrUser getUserByIdOrEmail(String userId, String userEmail) {
```

Comment:
If you allow either userId or userEmail to be null then you can implement both getUserId and getUserByEmail below with this function.

Reply:
I would then have to make getUserByIdOrEmail public since getUserByEmail is used in the OrganizationUpdater class. I decided to keep getUserByIdOrEmail as a separate (private) method and keep the other two public functions. In one of my discussions with Rob, we agreed that it would be useful to keep getUserById for a good API.